xcode/Subconscious/Shared/Components/Common/SubtextTextViewRepresentable.swift

```diff
@@ -164,38 +164,55 @@ struct SubtextTextViewRepresentable: UIViewRepresentable {
         }
     }
 
-    class SubtextTextParagraph: NSTextParagraph {
-        private(set) var subtext: Subtext
-
-        init(attributedString: NSAttributedString) {
-            SubtextTextViewRepresentable.logger.debug(
-                "SubtextTextParagraph: parse and render markup"
-            )
-            let subtext = Subtext(markup: attributedString.string)
-            let renderedAttributedString = subtext.renderAttributedString(
-                url: SubtextTextViewRepresentable.toSubURL
-            )
-            self.subtext = subtext
-            super.init(attributedString: renderedAttributedString)
-        }
-    }
-
     //  MARK: Coordinator
-    class Coordinator: NSObject, UITextViewDelegate, NSTextContentStorageDelegate, NSTextContentManagerDelegate, NSTextLayoutManagerDelegate {
+    class Coordinator: NSObject, UITextViewDelegate, NSTextContentStorageDelegate, NSTextStorageDelegate, NSTextContentManagerDelegate, NSTextLayoutManagerDelegate {
         /// Is event happening during updateUIView?
         /// Used to avoid setting properties in events during view updates, as
         /// that would cause feedback cycles where an update triggers an event,
         /// which triggers an update, which triggers an event, etc.
         var isUIViewUpdating: Bool
         var representable: SubtextTextViewRepresentable
         var renderTranscludeBlocks: Bool = false
+        var subtext: Subtext? = nil
```

Comment:
I'm caching the parse results so we can use them to detect transclude blocks without repeating work.